### ISSUE TITLE
Game crashes when mining an emerald block with a netherite pickaxe (likely other picks too).

### ISSUE THREAD

**Issue by LucilleTea** [PalgamingHD/End-Portal-Creater#2]
Placing an emerald block then trying to mine it will crash the game. No mods loaded other than Portal Crafter and Fabric API (required).
[Crash report](https://gist.github.com/LucilleTea/ff300fce1537e9cc208c203a9626ff7d)

Originally reported here: https://github.com/CaffeineMC/sodium-fabric/issues/520

**Comment by WardenDrew** [comment]
I can confirm this happens when attempting to mine an emerald block of at all, even hitting it with an empty hand.
Running Fabric API 34.1

**Comment by PalgamingHD** [comment]
Uh ill have to check that out if I ever get back to this, there's probably alternatives at this point anyway. Thanks for reporting, sorry about the 2 year late response

**Comment by WardenDrew** [comment]
Yeah its been a hot minute since I commented here. I'm not using this mod in any packs currently but good luck on it!
